README.md

```diff
@@ -141,6 +141,25 @@ hosted in the typical Jenkins' plugin directory structure.
 Note that that when `source` is specified, the `version` and `plugin_url`
 parameters will have no effect on the plugin retrieval URL.
 
+#### Configuration file
+
+If you need to create a configuration file to go with a plugin you can do this by setting `manage_config => true` *and*
+setting `config_filename => mypluginconfig.xml` *and* setting `config_content => 'string containing xml content of the file'`.
```

Comment:
Bonus points for updating the README!  Could you line wrap the new text at 79/80 cols?